dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -12,9 +12,9 @@
 
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Dark">
-            <StaticResource x:Key="NavigationViewDefaultPaneBackground" ResourceKey="SystemChromeMediumColor" />
+            <StaticResource x:Key="NavigationViewDefaultPaneBackground" ResourceKey="SystemControlTransientAcrylicElementBrush" />
```

Comment:
I don't know whether we need to touch this file. Feels like this is providing twice the opportunity to avoid crashing down level...which may not be necessary? Or maybe we should define these brushes to use the acrylic resources in the RS1 dictionary (with the understanding that the resources will gracefully fall back to solid colors) and remove their definitions from the NavigationView RS2 dictionary?

Reply:
Good point, since we are providing the same resource for RS1 and higher, defining it only in the RS1 file should be fine.

Reply:
Techincally, WinUI is only RS2 and greater now. So.... I'm actually not sure what happens to these files.